src/rpc/RPCHelpers.cpp

```diff
@@ -798,14 +799,10 @@ traverseOwnedNodes(
                         .count()
```

Comment:
Looks like this is also a case we can fix to use the new utility

Reply:
Search for `end - start` and also for `duration_cast<` to see if you can find even more places :+1:

Reply:
This one is tricky. We could not put the code into a lambda . The code return the whole function in the middle. I suggest we leave these tricky ones when we have unittest covered , or at least we know how to test it case by case.